src/main/java/com/mixpanel/android/mpmetrics/MixpanelPushNotification.java

```diff
@@ -60,19 +61,27 @@ public Notification createNotification(Intent inboundIntent) {
                 setTicker(data.message).
                 setContentIntent(contentIntent);
 
+        maybeSetSubTitle(data);
         maybeSetNotificationBarIcon(data);
         maybeSetLargeIcon(data);
         maybeSetExpandableNotification(data);
         maybeSetCustomIconColor(data);
         maybeAddActionButtons(data);
         maybeSetChannel();
+        maybeSetNotificationBadge(data);
 
         final Notification n = buildNotification();
         n.flags |= Notification.FLAG_AUTO_CANCEL;
 
         return n;
     }
 
+    protected void maybeSetSubTitle(NotificationData data) {
```

Comment:
should this be `maybeSetSubText`? Does calling `builder.setSubText` result in that text looking like a subtitle? My impression was it goes beneath the message body.